version.xml

```diff
@@ -1,7 +1,7 @@
 <!-- This file may be overwritten by automation. -->
 <Project>
   <PropertyGroup>
-    <KoreBuildChannel>dev</KoreBuildChannel>
+    <KoreBuildChannel>rel/2.0.0</KoreBuildChannel>
```

Comment:
We should remove SignalR from the "Universe" build. aspnet/Universe is meant to build a coherent set of packages that will ship together. With this change, SignalR is not going to fit that description.